package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
Is `jsnext:main` supposed to be full ES6/ES-next or only ES5+ES6-module-syntax? I'm not sure any more.